zkevm-circuits/src/evm_circuit/util/constraint_builder.rs

```diff
@@ -507,10 +507,6 @@ impl<'a, F: Field> ConstraintBuilder<'a, F> {
         is_code: Expression<F>,
     ) {
         let is_root_create = self.curr.state.is_root.expr() * self.curr.state.is_create.expr();
-        self.add_constraint(
```

Comment:
this is used by byte code lookup, in the root create case, not determined to lookup bytecode table or tx call data directly yet . can not remove now.

Reply:
I have a draft PR that addes the contract bytecode for this case to the bytecode table.